airsonic-main/src/main/java/org/airsonic/player/service/metadata/MetaData.java

```diff
@@ -152,12 +152,11 @@ public void setMusicBrainzReleaseId(String musicBrainzReleaseId) {
         this.musicBrainzReleaseId = sanitize(musicBrainzReleaseId);
     }
 
-    // filter out any \0 characters
     private String sanitize(String meta) {
-        if (meta != null) {
-            return meta.replace("\u0000", "");
-        } else {
+        if (meta == null || meta.replace("\u0000", "").length() == 0) {
             return null;
+        } else {
+            return meta.replace("\u0000", ":");
```

Comment:
Colon is pretty commonly used in conventional titles, especially for classical music.  I would strongly suggest using semi-colon as that is more effective as a multi-value separator.

Reply:
Or a dash, or a double colon, or semicolon+space,... make it configurable in the _advanced_ section. 

A quick survey over a set of 200.000 files (ranging from music to lectures) gives the following:

character(s) | count
-----|----
:|3457
/|1716
;|369
+|79
\\ |4
::|0

Double-colon to the rescue